swagger.yaml

```diff
@@ -0,0 +1,249 @@
+swagger: "2.0"
+
+info:
+  title: "Tweet API"
+  description: "This provides an API for creating a message (post), reading a message (get), editing a message (put), delete a message (delete)."
+  version: "1.0"
+
+host: localhost
+
+tags:
+- name: "tweets"
+
+schemes:
+  - "https"
+
+consumes:
+  - application/json
+produces:
+  - application/json
+
+paths:
+
+  /tweets:
+  
+    get:
+      tags:
+      - tweets
+      description: "search for a user's tweets"
+      produces:
+      - application/json
+
+      parameters:
+      - in: "query"
+        name: userName
+        required: false
+        type: string
+
+      - in: "query"
+        name: tweetId
+        required: false
+        type: integer
+
+      responses:
+        200:
+          description: Successful response
+          schema:
+            $ref: "#/definitions/tweetsData"
+        400:
+          description: The specified user ID is invalid
+          schema:
+            $ref: "#/definitions/Error"
+        404:
+          description: page not found
+          schema:
+            $ref: "#/definitions/Error"
+        405:
+          description: Invalid input
+          schema:
+            $ref: "#/definitions/Error"
+        default:
```

Comment:
When will you return the default error? Could you explicitly define the error code like [here](https://github.com/osu-mist/slay-the-spire-cards-api/blob/master/swagger.yaml#L121-L124)?